sigridci/sigridci/report.py

```diff
@@ -76,7 +76,9 @@ def getSummaryText(self, feedback, options):
             return f"✅  You wrote maintainable code and achieved your Sigrid objective of {target}"
         elif status == ObjectiveStatus.IMPROVED:
             return f"↗️  You improved your code's maintainability towards your Sigrid objective of {target}"
+        elif status == ObjectiveStatus.UNCHANGED:
+            return "⏸️️  You didn't improve your code, but it also didn't get any worse"
         elif status == ObjectiveStatus.WORSENED:
-            return f"❌  Your code did not manage to improve towards your Sigrid objective of {target}"
+            return f"⚠️  Your code did not manage to improve towards your Sigrid objective of {target}"
```

Comment:
I don't think the "manage to" bit is needed, and in my ears it sounds overly negative. This would be the more neutral alternative:
```suggestion
            return f"⚠️  Your code did not improve towards your Sigrid objective of {target}"
```